Sqlite reads TCL config from /usr/lib/tclConfig.sh. That lists
/opt/SUNWspro/bin/cc as a compiler which no longer exists in our CBE. This
change is overwriting the config to set the compiler to gcc.

This is interim change until we move TCL compilation to gcc too.

This is solaris specific patch, not suitable for upstream.

--- sqlite-autoconf-3210000/tea/tclconfig/tcl.m4	2017-11-15 08:52:28.206926381 +0000
+++ sqlite-autoconf-3210000/tea/tclconfig/tcl.m4	2017-11-15 08:52:24.892793953 +0000
@@ -360,6 +360,8 @@ AC_DEFUN([TEA_LOAD_TCLCONFIG], [
         AC_MSG_RESULT([could not find ${TCL_BIN_DIR}/tclConfig.sh])
     fi
 
+    TCL_CC=/usr/gcc/5/bin/gcc
+
     # eval is required to do the TCL_DBGX substitution
     eval "TCL_LIB_FILE=\"${TCL_LIB_FILE}\""
     eval "TCL_STUB_LIB_FILE=\"${TCL_STUB_LIB_FILE}\""
